packages/cli/src/evaluation/test-runner/test-runner.service.ee.ts

```diff
@@ -157,6 +159,14 @@ export class TestRunnerService {
 		const evaluationWorkflow = await this.workflowRepository.findById(test.evaluationWorkflowId);
 		assert(evaluationWorkflow, 'Evaluation workflow not found');
 
+		// 0. Create new Test Run
+		const testRun = this.testRunRepository.create({
+			testDefinition: { id: test.id },
+			status: 'new',
+		});
+
+		await this.testRunRepository.save(testRun);
```

Comment:
Could we use `insert` instead or does `.save` update something behind the scenes that we need?

Reply:
We have to use `.save`, because the entity uses `WithTimestampsAndStringId` mixin, which rely on `@BeforeInsert` hook to generate string ID.

Reply:
But I moved this code to the separate method in repo